app/src/main/java/changhwan/experiment/sopthomework/BindingAdapters.kt

```diff
@@ -4,13 +4,18 @@ package changhwan.experiment.sopthomework
 import android.graphics.drawable.Drawable
 import android.widget.ImageView
 import androidx.databinding.BindingAdapter
+import androidx.lifecycle.MutableLiveData
+import com.bumptech.glide.Glide
 
 
 object BindingAdapters {
 
     @JvmStatic
-    @BindingAdapter("imgRes")
-    fun setImage (imageview : ImageView, resid : Int){
-        imageview.setImageResource(resid)
+    @BindingAdapter("recyclerGlide")
+    fun setImage (imageview : ImageView, url : MutableLiveData<String>){
+        Glide.with(imageview.context)
+            .load(url.value)
+            .circleCrop()
+            .into(imageview)
     }
```

Comment:
바인딩 어댑터의 경우 확장함수로

```suggestion
  @BindingAdapter("recyclerGlide")
      fun ImageView.setImage (url : MutableLiveData<String>){
          Glide.with(context)
              .load(url.value)
              .circleCrop()
              .into(imageview)
      }
```
이렇게 쓰셔도 됩니다